datastore/query.go

```diff
@@ -625,6 +626,70 @@ func (c *Client) Count(ctx context.Context, q *Query) (n int, err error) {
 	}
 }
 
+const (
+	// QueryModeNormal is the default mode. Only the query results are returned.
+	QueryModeNormal QueryMode = iota
+	// QueryModeExplain returns only the query plan, without any results or execution
```

Comment:
apologies if I'm missing - how do these map to NORMAL, PLAN, AND PROFILE?

Reply:
QueryModeExplain => PLAN
QueryModeExplainAnalyze => PROFILE
This line converts QueryMode to proto: https://github.com/googleapis/google-cloud-go/pull/9200/files#diff-c04af33c4c45ff4606abe36bd859432effb278cca4146d3f818e540f0479fd8dR652

Reply:
please add a comment for each value with the appropriate numeric value

Reply:
Added

Reply:
It might also be useful to mention the enum value this maps to ie. PLAN, PROFILE.

Reply:
The protos have been changed. no more enums